markdown/3.0/en/ff-paging-dropdown.md

```diff
@@ -26,12 +26,32 @@ When the attribute `show-selected` is set on the `ff-paging-dropdown`, the dropd
 
 In difference to an `ff-paging` element the `ff-paging-dropdown` accepts only one `ff-paging-item` as a template for all generated pages. 
 
-**NOTE:** All `ff-paging-item`s generated by the `ff-paging-dropdown` element will have a
-type attribute like `&lt;ff-paging-item type=&quot;pageLink&quot;&gt;` except the currently selected one.
-The selected one will have a type attribute like `&lt;ff-paging-item type=&quot;currentLink&quot;&gt;`
 
 ```html
 <ff-paging-dropdown show-selected>
     <ff-paging-item>{{caption}}</ff-paging-item>
 </ff-paging-dropdown>
+```
+
+### Rendered HTML
```

Comment:
I think that it somehow refers to this sentence: ``In difference to an `ff-paging` element the `ff-paging-dropdown` accepts only one `ff-paging-item` as a template for all generated pages.`` from the previous section.

Reply:
It also applies to the examples in the **Overview** section.  That's why I feel it could easily be a top level section. Kind of a stand-alone reference.
You decide. Let's not spend too much time on a topic a full-time technical writer would probably rewrite anyway ^^

Reply:
Is it?
Leave it if you think it's more suitable in this location.